pdns/dnsdist.cc

```diff
@@ -748,6 +749,22 @@ bool processQuery(LocalStateHolder<NetmaskTree<DynBlock> >& localDynNMGBlock,
     g_rings.queryRing.push_back({now,*dq.remote,*dq.qname,dq.len,dq.qtype,*dq.dh});
   }
 
+  if(g_qcount.enabled) {
+    WriteLock wl(&g_qcount.queryLock);
+    string qname = (*dq.qname).toString(".", false);
```

Comment:
Just a nit, but for a better readability we have `toStringNoDot()` that does `toString(".", false)`